docs/source/models/gbd2017/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -47,27 +47,70 @@ represents **all** forms of anemia that are attributable to several causes,
 including causes such as hemoglobinopathies and hemolytic anemias that are not 
 causally attributable to iron deficiency. 
 
-Iron Deficiency Risk Factor
-+++++++++++++++++++++++++++
-
-The iron deficiency risk factor in GBD 2017 is modeled as a distribution of 
-hemoglobin concentrations. The risk factor is a **continuous measure**.
-
-.. todo::
-
-	Describe the population and distribution type (normal vs. ensemble)
+Anemia Impairment
++++++++++++++++++
+
+The anemia impairment in GBD 2017 represents the total prevalence of anemia due 
+to all causes modeled in GBD (ex: dietary iron deficiency anemia, anemia due to 
+maternal hemorrhage, sickle cell anemia, etc.). Estimating the total prevalence 
+of the anemia impairment for a given population is the first step in modeling 
+anemia in GBD 2017. This is done by fitting a distribution of hemoglobin levels 
+for that population from primary input data. For GBD 2017, an ensemble 
+distribution was used, which was 40% gamma and 60% mirror gumbel. Source code 
+for this process is available `here <https://stash.ihme.washington.edu/projects/MNCH/repos/anemia/browse/model/envelope>`_.
+
+Once a the distribution is fit to hemoglobin levels for a particular age-, 
+sex-, and location-specific demographic group, age- and sex-specific hemoglobin 
+thresholds, as defined by the WHO and described in the table below, are used to 
+determine the prevalence of anemia by severity for that population.
 
-Modeling Strategy for the Iron Deficiency Risk Factor
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. list-table:: Hemoglobin Thresholds (g/L)
+	:widths: 15, 15, 15, 15
+	:header-rows: 1
 
-.. todo::
+	* - Group
+	  - Mild Anemia
+	  - Moderate Anemia
+	  - Severe Anemia
+	* - Males and Females <1 month
+	  - 130-149
+	  - 90-129
+	  - <90
+	* - Males and Females 1 month - 5 years
+	  - 100-109
+	  - 70-99
+	  - <70
+	* - Males and Females 5-14 years
+	  - 110-114
+	  - 80-109
+	  - <80
+	* - Males 15+ years
+	  - 110-129
+	  - 80-109
+	  - <80
+	* - Females 15+ years, non-pregnant 
+	  - 110-119
+	  - 80-109
+	  - <80
+	* - Females 15+ years, pregnant
+	  - 100-109
+	  - 70-00
+	  - <70
 
-	Describe anemia envelope and causal attribution
+The prevalence of anemia as calculated in the process described above serves as 
+the overall anemia envelope for a age-, sex-, and location-specific demographic 
+groups and prevanet cases of anemia in the anemia envelope are then causally 
+attributed to various causes in GBD 2017 that have anemia as seqeulae. This is 
+done through a process described in the GBD 2017 non-fatal methods appendix.
```

Comment:
This process is *insufficiently* described in the YLD appendix... I'd like to eventually get a more explicit description of their actual process.

Reply:
I added a to-do box for this in the latest commit. This might be something that we need to sit down with the anemia modelers to figure out in detail